specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2022-10-01/common-types.json

```diff
@@ -32,40 +97,66 @@
         }
       }
     },
-    "ProxyResource": {
-      "description": "The Resource model definition.",
+    "Operation": {
+      "description": "A Cost management REST API operation.",
       "type": "object",
       "properties": {
         "id": {
-          "readOnly": true,
+          "description": "Operation id: {provider}/{resource}/{operation}.",
           "type": "string",
-          "description": "Resource Id."
+          "readOnly": true
         },
         "name": {
-          "readOnly": true,
+          "description": "Operation name: {provider}/{resource}/{operation}.",
           "type": "string",
-          "description": "Resource name."
+          "readOnly": true
         },
-        "type": {
+        "display": {
+          "description": "The object that represents the operation.",
+          "type": "object",
+          "properties": {
+            "provider": {
+              "description": "Service provider: Microsoft.CostManagement.",
+              "type": "string",
+              "readOnly": true
+            },
+            "resource": {
+              "description": "Resource on which the operation is performed: Dimensions, Query.",
+              "type": "string",
+              "readOnly": true
+            },
+            "operation": {
+              "description": "Operation type: Read, write, delete, etc.",
+              "type": "string",
+              "readOnly": true
+            },
+            "description": {
+              "description": "Operation description",
+              "type": "string",
+              "readOnly": true
+            }
+          }
+        }
+      }
+    },
+    "OperationListResult": {
```

Comment:
>OperationListResult [](http://example.com/codeflow?start=5&length=19)

Did you check your definitions match the ones in common-types for all their properties? Please do.

Reply:
It saves you from S360 violations etc.